packages/one-app-runner/src/startApp.js

```diff
@@ -91,6 +93,18 @@ module.exports = async function startApp({
   };
 
   const generateModuleMap = () => (moduleMapUrl ? `--module-map-url=${moduleMapUrl}` : '');
+
+  if (createDockerNetwork && dockerNetworkToJoin) {
```

Comment:
should provide some feedback if `createDockerNetwork` is `true` but no `dockerNetworkToJoin`

Reply:
"createDockerNetwork flag is true but dockerNetworkToJoin is undefined, please pass a valid network name value"